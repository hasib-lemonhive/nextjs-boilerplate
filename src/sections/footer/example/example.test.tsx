import { render } from '@testing-library/react';
import ExampleFooter from '.';

it('Renders Example Footer correctly', () => {
  const { getByTestId } = render(<ExampleFooter testId="footer" />);
  const element = getByTestId('footer');
  expect(element).toBeInTheDocument();
  expect(element).toMatchSnapshot();
});
