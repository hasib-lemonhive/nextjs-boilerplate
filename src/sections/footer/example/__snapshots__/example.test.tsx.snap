// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Example Footer correctly 1`] = `
<section
  class="dark:bg-slate-900 bg-gray-400 flex items-center justify-center py-6"
  data-testid="footer"
>
  <p>
    Footer
  </p>
</section>
`;
